presentation/common/src/presentation-common/PresentationRpcInterface.ts

```diff
@@ -133,7 +134,7 @@ export class PresentationRpcInterface extends RpcInterface {
   public static readonly interfaceName = "PresentationRpcInterface"; // eslint-disable-line @typescript-eslint/naming-convention
 
   /** The semantic version of the interface. */
-  public static interfaceVersion = "2.9.0";
+  public static interfaceVersion = "2.10.0";
```

Comment:
Change in minor version indicates a method was added, but I don't see a new method. You did change the format in which arrays of Ids can be passed though, right? That seems like it would potentially cause older frontends talking to newer GPB to malfunction?

Reply:
Yeah, I bumped the version to avoid new frontends talking to old backends (thus backends not understanding what frontends are sending), but completely missed the fact that new backends might get requests from old frontends, which means they have to understand the old format as well. Fixed that.